metric/metric.go

```diff
@@ -23,6 +23,10 @@ const (
 	SheetCountMetricName metric.Name = "bb.sheet.count"
 	// MemberCountMetricName is the metric name for member count
 	MemberCountMetricName metric.Name = "bb.member.count"
+	// OpenAPIMetricName is the metric name for OpenAPI
+	OpenAPIMetricName metric.Name = "bb.api.call"
```

Comment:
We can use the current method for now. However, I think API telemetry (online) is different from resource usage reporting (offline, current metric dashboard). The sql/advise API has low traffic which should fit the current telemetry pipeline. However, the api/call might need other types of pipeline (GCP Cloud Monitoring API maybe) later.